src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        String color = colorSupplier.getRandomColor().toString();
+        int number = getRandomNumber();
+        return new Ball(color, number);
+    }
+
+    private int getRandomNumber() {
+        return random.nextInt(101);

```

Comment:
Please, follow [this](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants) rule.

Reply:
It would be good to put the maximum ball number in a separate variable and then add `1` to the random number to get the expected result.
```
private static final int MAX_NUMBER = 100;
....
return random.nextInt(MAX_NUMBER) + 1;
```
In this way, the number 100 will be stored in a good format and the question `Why it is 101 and not 100?` will not appear.